src/models/device.ts

```diff
@@ -157,6 +157,15 @@ const getDeviceModel = function (
 	});
 	/* eslint-enable @typescript-eslint/no-require-imports */
 
+	const getOsUpdateHelperv2 = once(async () => {
+		const $deviceUrlsBase = await getDeviceUrlsBase();
+		const { getOsUpdateHelperv2: _getOsUpdateHelperv2 } =
+			require('../util/device-actions/os-update') as typeof import('../util/device-actions/os-update');
```

Comment:
I think you can use a normal import like:
```suggestion
		const { getOsUpdateHelperv2: _getOsUpdateHelperv2 } = await('../util/device-actions/os-update');
```
since we're already in an async context and don't need to worry about using require for synchronous loading